logstash-core/src/main/java/org/logstash/instrument/metrics/gauge/LazyDelegatingGauge.java

```diff
@@ -71,6 +73,11 @@ public Object get() {
         return lazyMetric == null ? null : lazyMetric.get();
     }
 
+    @SuppressWarnings("rawtypes")
+    public Optional getMetric() {
+        return Optional.of(lazyMetric);
```

Comment:
`Optional::of(null)` will [throw an NPE](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html#of(T))

```suggestion
        return Optional.ofNullable(lazyMetric);
```

Reply:
Thanks for catching this!